src/DwhProvider/RemoteSnowflakeProvider.php

```diff
@@ -33,5 +33,13 @@ public function createDbtYamlFiles(array $configurationNames = []): void
             $this->getOutputs($configurationNames, static::getDbtParams())
         );
         $this->setEnvVars();
+
+        $this->logger->info($this->getConnectionLogMessage());
```

Comment:
Nemělo by tohle být i u ostatních providerů?

Reply:
To se zdedi z tohodle parenta do childu

Reply:
jo, už to vidím 👍  No je to trochu divný, možná by měl bejt nějaký common provider nebo tak, než extendovat ten snflk. Ale to je jedno.

Reply:
Jasne no, jsem tam to dedeni ani nechtel moc davat :D To este poladime. Idealne tu spolocnu funkcionalitu vytiahnut do nejakej hleper classu alebo traitu